src/main/java/axiomq/com/starwars/controllers/UserController.java

```diff
@@ -17,11 +18,13 @@ public class UserController {
 
     private UserService userService;
 
+    @PreAuthorize("hasRole('ROLE_ADMIN')")
```

Comment:
I think that Branimir told us that we should keep authorization in one place. For example in SecurityConfig simply create new antMatcher for this endpoint.